README.md

```diff
@@ -273,6 +273,23 @@ Tags specific for this measurement:
 | commit_messages | string | Comma-separated list of commit messages | |
 | culprits | string | Comma-separated list of commit authors | |
 
+#### `junit_data` (since 2.3)
+
+| Metric | Type | Description | Introduced in |
+| --- | --- | --- | --- |
+| suite_name | string | Testsuite name | 2.3 |
+| test_name | string | Test name | 2.3 |
+| test_status | string | PASSED, SKIPPED, FAILED, FIXED, REGRESSION | 2.3 |
+| test_status_ordinal | integer | 0-4 in order of test_status | 2.3 |
+
+Tags specific for this measurement:
+
+| Tag | Description | Introduced in |
+| --- | --- | --- |
+| suite_name | Testsuite name | 2.3 |
+| test_name | Test name | 2.3 |
+| test_status | Test result | 2.3 |
```

Comment:
The measurement is already introduced in version 2.3 (presumably). I don't think it's necessary to repeat `2.3` on every row. Mainly the idea is that if something is added to a metric in a later version, it's easy to see when specific metrics were introduced within a specific measurement (like `sonarqube_data` was added already in 1.11, but I've only mentioned the new metrics in 2.2).